tools/check_copyright_and_license_disclaimers.py

```diff
@@ -0,0 +1,80 @@
+#! /usr/bin/env python
+#
+# Copyright © 2022 Intel Corporation
+#
+# SPDX-License-Identifier: MIT
+
+import argparse
+import os
+import sys
+
+
+COPYRIGHT_AND_LICENSE_DISCLAIMER = "# Copyright © 2022 Intel Corporation\n#\n# SPDX-License-Identifier: MIT\n"
+
+def file_has_disclaimer(filename, verbose=False):
+  if verbose:
+    print(f"Checking: {filename}")
+  with open(filename) as fp:
+    # NOTE: allow shebang+empty line before
+    header = "".join(fp.readline() for _ in range(5))
+  return COPYRIGHT_AND_LICENSE_DISCLAIMER in header
+
+
+def check_all(directory, verbose=False):
+  missing = []
+  for dirpath, dirnames, filenames in os.walk(directory):
+    for filename in filenames:
+      absolute_filename = os.path.join(dirpath, filename)
+      if is_file_relevant(absolute_filename):
+        if not file_has_disclaimer(absolute_filename, verbose=verbose):
+          missing.append(absolute_filename)
+  return missing
+
+
+def is_file_relevant(filename):
+  return filename.endswith((".py", "Dockerfile")) and os.stat(filename).st_size > 0
```

Comment:
skipping `st_size == 0` allows us to have totally empty `__init__.py`, which seems consistent with other repos